src/toolkit.cpp

```diff
@@ -818,12 +818,13 @@ bool Toolkit::LoadData(const std::string &data, bool resetLogBuffer)
     m_doc.PrepareData();
     m_doc.InitSelectionDoc(m_docSelection, true);
 
+    if (m_options->m_mensuralScoreUp.GetValue()) {
+        m_doc.ScoringUpDoc();
+        //m_doc.ConvertToCastOffMensuralDoc(true);
+    }
+
```

Comment:
Why are the test file not working? They should be detected as mensural music, shouldn't they?

Reply:
They are actually detected as mensural, but they are not processed for some reason (if I remember correctly). I will play with it a little again and if I cannot figure it out, I will send you a file so we can look at this together.

Reply:
It works! Will push the change. I don't know what happened. I tried everything again now (with the scoring up in the right place) and things are looking good.

Now the only thing missing would be for me to prepare a file to test changes in the mensuration, debug if needed, and start putting a test suite in the right place (I have files for that ready, just missing the one for changes in mensuration).